config/crd/bases/policies.kubewarden.io_policyservers.yaml

```diff
@@ -1,9 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
I would prefer to generate this manifests with controller-tools at 0.8.0 at least (05 Jan 2022). 0.4.1 goes back to 17 Nov 2020.

Reply:
done in https://github.com/kubewarden/kubewarden-controller/pull/247/commits/f71b653c1b969cecaae9a1c1c1cbdc3e3e88c73f (I had an old version in my bin folder and make controller-gen did nothing until I removed it first)